app/responders/sections_responder.rb

```diff
@@ -0,0 +1,9 @@
+class SectionsResponder < ApplicationResponder
+  @@sections_topic_name = ENV['SECTIONS_TOPIC_NAME']
```

Comment:
use https://apidock.com/ruby/String/to_sym and remove interpolation

`@@topic_name = ENV['SECTIONS_TOPIC_NAME'].to_sym`

Reply:
I've made the change. Nevertheless, can you please clarify by what you mean by interpolation in this context, please?

Reply:
http://ruby-for-beginners.rubymonstas.org/bonus/string_interpolation.html